common/define-grammar.js

```diff
@@ -1214,7 +1242,7 @@ module.exports = function defineGrammar(dialect) {
         ),
       ),
 
-      _argument_name: $ => seq(
+      _argument_name: $ => reserved('properties', seq(
```

Comment:
yeah, the PR is currently adding support by default for **non-reserved** keywords to be usable as the word token if they overlap (the pattern for the word token matches the keyword in its entirety). So obviously operators wouldn't match, but any keywords like these would :) Really excited for that since a lot of grammars will no longer need this "_reserved_identifier" hack they currently have.